flocker/common/functional/test_script.py

```diff
@@ -135,12 +135,10 @@ def test_stdout_stderr(self):
         def got_messages(messages):
             assertContainsFields(self, messages[1],
                                  {u"message_type": u"twisted:log",
-                                  u"message": u"stdout!",
-                                  u"error": False})
+                                  u"message": u"stdout!"})
             assertContainsFields(self, messages[2],
```

Comment:
This probably isn't a difference that really matters to us.  The change in Twisted is a regression, though, so we should probably expect a 15.2.1 that re-introduces the previous behavior.  If that happens we won't want to have changed our tests.